### PR TITLE
feat(PageHero): add option to show image on mobile

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Previously, the image was always hidden on mobile. This adds the ability to choose whether to show the image on mobile or not with an optional boolean prop, the default being the previous standard to hide on mobile.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/J5yB1jyELiNt3Vb47T9GcE/Job-Readiness?node-id=2%3A43
- [x] Related to JIRA ticket: [REACH-1325]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.90-alpha.23fdc2.0<br/>@codecademy/gamut-labs@23.0.1-alpha.23fdc2.0<br/>@codecademy/styleguide@46.0.1-alpha.23fdc2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614b4ac4405d3ae81455e6ca--gamut-preview.netlify.app>https://614b4ac4405d3ae81455e6ca--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614b4ac4405d3ae81455e6ca>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.90-alpha.908b84.0<br/>@codecademy/gamut-labs@23.0.1-alpha.908b84.0<br/>@codecademy/styleguide@46.0.1-alpha.908b84.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614b5031da0f9403e3aaf0a9--gamut-preview.netlify.app>https://614b5031da0f9403e3aaf0a9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614b5031da0f9403e3aaf0a9>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Review by aionate0812 — APPROVED** [review]
lgtm - TIL about `initial`

**Review by MattSchiller — APPROVED** [review]
:shipit:

File: packages/gamut-labs/src/LandingPage/PageHero.tsx [hunk 1]
```diff
@@ -91,13 +93,18 @@ export const PageHero: React.FC<PageHeroProps> = ({
...
+        >
           {media.type === 'image' ? (
             <Image
               src={media.src}
               alt={media.alt}
               width={1}
-              display={{ _: 'none', sm: 'initial' }}
+              display={
+                showImageOnMobile ? undefined : { _: 'none', sm: 'initial' }
+              }
```
Comment (codecaaron) [inline packages/gamut-labs/src/LandingPage/PageHero.tsx:107]: NITTIEST NIT OF ALL TIME

```suggestion
              display={{
                 _: showImageOnMobile ? 'initial' : 'none' , 
                 sm: 'initial' 
               }}
```

**Review by codecaaron — APPROVED** [review]

File: packages/gamut-labs/src/LandingPage/PageHero.tsx (hunk 1 above)
Comment (judahtanthony) [reply packages/gamut-labs/src/LandingPage/PageHero.tsx:107]: Hi Aaron, I ported this suggestion into a new branch https://github.com/Codecademy/client-modules/pull/2085/files

File: packages/gamut-labs/src/LandingPage/PageHero.tsx [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
 import { CTA } from './CTA';
 import { Description } from './Description';
-import { PageHeroMedia } from './PageHeroMedia';
 import { Title } from './Title';
 import { BaseProps } from './types';
 
-const LeftColumn = styled(Column)`
-  align-content: center;
-`;
```
Comment (MattSchiller) [inline packages/gamut-labs/src/LandingPage/PageHero.tsx:13]: @judahtanthony Looks like there's some clobbered changes in here?

**Comment by judahtanthony** [comment]
superseded by https://github.com/Codecademy/client-modules/pull/2085

**Pull request closed without merging**